### PR TITLE
Sunset rest.wikimedia.org

### PR DESCRIPTION
**Pull request by gwicke**
Start returning errors, but provide a friendly hint in the error details.

### CODE REVIEW

File: lib/normalize_title_filter.js [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
-            return origNext(hyper, req)
-            .then(function(res) {
-                if (res) {
-                    res.headers = res.headers || {};
-                    res.headers['cache-control'] = 'no-cache';
-                }
-                return res;
-            });
-        };
+        hyper.log('warn/normalize/rest_wikimedia_org', {
```
Comment (Pchelolo) [inline lib/normalize_title_filter.js]: I think it's better to still sample the logs 1:10

**Comment by Pchelolo** [comment]
:sunrise: :wave: rest.wikimedia.org 👍

File: lib/normalize_title_filter.js (hunk 1 above)
Comment (d00rman) [inline lib/normalize_title_filter.js]: :+1: at least until we know that the volume is low

File: lib/normalize_title_filter.js [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
+            msg: 'Request to rest.wikimedia.org',
+        });
+
+        // Satisfy jscs's 100 char limit
+        var mURL = 'https://lists.wikimedia.org/pipermail/mediawiki-api/2016-January/003691.html';
+        throw new HTTPError({
+            status: 410,
+            body: {
+                type: 'not_found',
```
Comment (d00rman) [inline lib/normalize_title_filter.js]: `s/not_found/gone/` ?

Comment (gwicke) [inline lib/normalize_title_filter.js]: I originally used not_found, as we already have an error page for that at https://www.mediawiki.org/wiki/HyperSwitch/errors/not_found. I agree that `gone` is more precise though, so I went ahead & created https://www.mediawiki.org/wiki/HyperSwitch/errors/gone / changed the type.

**Pull request merged**